<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchPage</class>
 <widget class="QWidget" name="SearchPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>邮件数据搜索</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_target">
          <property name="text">
           <string>选择检索位置：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="button_flash">
            <property name="text">
             <string>刷新</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="8" column="0">
            <widget class="QLabel" name="label_filename">
             <property name="text">
              <string>附件文件名关键字：</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_subject">
             <property name="text">
              <string>主题关键字：</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QLineEdit" name="lineEdit_file_content"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_cc">
             <property name="text">
              <string>抄送人邮箱地址：</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_account">
             <property name="text">
              <string>所属邮箱账户：</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEdit_to"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_account"/>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_file_content">
             <property name="text">
              <string>附件内容关键字：</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_to">
             <property name="text">
              <string>收件人邮箱地址：</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_body">
             <property name="text">
              <string>正文关键字：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_from">
             <property name="text">
              <string>发件人邮箱地址：</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lineEdit_subject"/>
           </item>
           <item row="5" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QDateEdit" name="dateEdit_start"/>
             </item>
             <item>
              <widget class="QLabel" name="label_date">
               <property name="text">
                <string>至</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="dateEdit_end"/>
             </item>
            </layout>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_from"/>
           </item>
           <item row="5" column="0">
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>全部收件日期</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>指定收件日期</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>指定日期及以前</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>指定收件时段</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="lineEdit_body"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_bcc">
             <property name="text">
              <string>密送人邮箱地址：</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QLineEdit" name="lineEdit_filename"/>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_bcc"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_cc"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textBrowser_details">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="button_clear">
            <property name="text">
             <string>重置搜索</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="button_search">
            <property name="text">
             <string>开始搜索</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
